janusgraph-core/src/main/java/org/janusgraph/core/ConfiguredGraphFactory.java

```diff
@@ -105,7 +105,7 @@ public static JanusGraph open(String graphName) {
         final Map<String, Object> graphConfigMap = configManagementGraph.getConfiguration(graphName);
         Preconditions.checkState(null != graphConfigMap,
                                 "Please create configuration for this graph using the ConfigurationManagementGraph#createConfiguration API.");
-        final JanusGraphManager jgm = JanusGraphManager.getInstance();
+        final JanusGraphManager jgm = JanusGraphManagerUtility.getInstance();
```

Comment:
Update this one too

https://github.com/dpitera/janusgraph/blob/b22c604a172940816fa2a60299e55f5a41724833/janusgraph-core/src/main/java/org/janusgraph/core/ConfiguredGraphFactory.java#L135